src/index.js

```diff
@@ -27,7 +27,7 @@ app.use(morgan('combined'));
 
 // endpoint to return all to-dos
 app.get('/', async (req, res) => {
-  res.send(await getToDos());
+  res.send(await getToDos(req));
```

Comment:
if you do need to know if there is another page or not, there are better ways other than counting how many objects there are in the whole collection. For example, you could simply find `perPage + 1` then, if there was `perPage + 1` rows returned from the database, you would know that there is another page.